### PR TITLE
CLDC-2803 Show frequency in rent range soft validation pages

### PR DESCRIPTION
**Pull request by natdeanlewissoftwire**
new behaviour for 22/23 and 23/24 logs: image
ticket: https://dluhcdigital.atlassian.net/browse/CLDC-2803

### CODE REVIEW

File: app/views/logs/download_csv.html.erb [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <h1 class="govuk-heading-l">Download CSV</h2>
```
Comment (natdeanlewissoftwire) [inline app/views/logs/download_csv.html.erb:9]: unrelated fix

File: app/models/form/lettings/pages/max_rent_value_check.rb [hunk 2]
```diff
@@ -19,6 +19,6 @@ def questions
   end
 
   def interruption_screen_question_ids
-    %w[brent startdate uprn postcode_full la beds rent_type needstype]
+    %w[brent startdate uprn postcode_full la beds rent_type needstype period]
```
Comment (kosiakkatrina) [inline app/models/form/lettings/pages/max_rent_value_check.rb]: nice

**Review by kosiakkatrina — APPROVED** [review]
